### ISSUE TITLE
Warning: Detected that Rio cluster domain is not generated for this cluster right now

### ISSUE THREAD

**Issue by rancher-max** [rancher/rio#599]
This is just meant to inform that its waiting for the system to boot up. We should probably update the message to reflect that.

**Steps I Ran**
1. Created a new k3s cluster using Linode.
2. Added one additional worker to k3s cluster
3. Added this configuration into a Rancher setup and got the KUBECONFIG file from there
4. Set my local KUBECONFIG to use that.
5. `curl -sfL https://get.rio.io | INSTALL_RIO_VERSION=v0.6.0-alpha.1 sh -`
6. `rio install` 

This message popped up pretty quickly and has been sitting on it for some time (approx. 10 minutes so far)

**Comment by rancher-max** [comment]
That one took so long I had to cancel it. A different rio install worked, but had a bad feel to it:
`/ Warning: ClusterDomain is not accessible. Error: Get http://xxxxxx.on-rio.io:30630: dial tcp x.x.x.x:30630: connect: connection refused`

**Comment by izaac** [comment]
For me is hanging the gateway service:
```bash
k get pods -n rio-system --no-headers | grep "Pending\|Completed"
svclb-gateway-proxy-v2-8899m        0/2   Pending     0     5m55s
```

**Comment by benmoss** [comment]
I was trying to use `kind` for my cluster and seems to get this error every time.

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2. @rancher-max  please retest it, we have improved install user-experience on that.

**Comment by rancher-max** [comment]
Appears to be working on:
- k3s v0.10.0: `rio install`
- rke k8s 1.16: `rio install --ip-address <ip1>,<ip2>,...,<ipN>
- GKE: `rio install`
- EKS: `rio install`

And UX is better across the board. Warnings are gone.
Ran a few services in each of these clusters. The only one currently failing is EKS with `curl: (6) Could not resolve host: demo-v1-default.xxxxxx.on-rio.io`, which is not related to this issue. Will open a separate issue to handle that.

Closing this as passing in v0.6.0-alpha.2
